packages/react-sdk/src/components/html.tsx

```diff
@@ -0,0 +1,94 @@
+import {
+  forwardRef,
+  useContext,
+  useEffect,
+  useRef,
+  type ForwardedRef,
+} from "react";
+import { mergeRefs } from "@react-aria/utils";
+import { ReactSdkContext } from "../context";
+
+type Props = {
+  code: string;
+  executeScriptInCanvas: boolean;
+};
+
+type ChildProps = {
+  innerRef: ForwardedRef<HTMLDivElement>;
+  // code can be actually undefined when prop is not provided
+  code?: string;
+};
+
+/**
+ * Scripts are executed when rendered client side
+ */
+const ExecutableHtml = (props: ChildProps) => {
```

Comment:
i would add. comment that we don't have a server rendering inside builder canvas, so the range trick (give a link to the trick article) is used